src/main/java/rx/Single.java

```diff
@@ -1569,14 +1569,12 @@ public final void onNext(T args) {
      * @param subscriber
      *            the Subscriber that will handle the emission or notification from the Single
      */
-    public final void unsafeSubscribe(Subscriber<? super T> subscriber) {
+    public final Subscription unsafeSubscribe(Subscriber<? super T> subscriber) {
```

Comment:
Right right, I mean that we will have to release it as 1.2 instead of 1.1.x,  I guess. 

Maintainers: what do you think?